app/src/gui/components/notebook/add_record_by_type.tsx

```diff
@@ -25,15 +26,21 @@ type AddRecordButtonsProps = {
 export default function AddRecordButtons(props: AddRecordButtonsProps) {
   const {project} = props;
   const project_id = project.project_id;
-  const theme = useTheme();
-  const mq_above_md = useMediaQuery(theme.breakpoints.up('md'));
-  const mq_above_sm = useMediaQuery(theme.breakpoints.up('sm'));
+  const muiTheme = useTheme();
+  const mq_above_md = useMediaQuery(muiTheme.breakpoints.up('md'));
+  const mq_above_sm = useMediaQuery(muiTheme.breakpoints.up('sm'));
   const [uiSpec, setUiSpec] = useState<ProjectUIModel | undefined>(undefined);
   const [showQRButton, setShowQRButton] = useState(false);
   const [selectedRecord, setSelectedRecord] = useState<
     RecordMetadata | undefined
   >(undefined);
 
+  const envTheme = import.meta.env.VITE_THEME;
+
+  // Determine 'add new record' button label based on the environment theme
+  const buttonLabel =
+    envTheme === 'bubble' ? `Survey New ${RECORD_LABEL}` : 'New Record';
```

Comment:
Again, use the form name here.  I think 'New Site' is probably ok rather than 'Survey New Site' but we can inject text like this in the theme if we have to.

Reply:
I agree with your suggestion here, New site makes more logical sense than Survey new Site. Will update this.